### ISSUE TITLE
[Bug][Tabstrip] DropDowns rendered in initially hidden container shows the popup with 0px width

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-vue#478]
When adding a second MultiSelect in a Tabstrip, the component is rendered with an improper width. This can be seen in the following demo:

https://codesandbox.io/s/nostalgic-sky-j88yh4

Steps to reproduce:
1. Open the first tab and click on the MultiSelect, notice that it opens properly.
2. Select the second tab and click on the MultiSelect, notice that it is opening with a small width and is messed up.
 
Expected behavior
The second MultiSelect should open as well

Workaround:
The only known workaround at the moment is to add a v-if statement so that the component is re-rendered: 

https://codesandbox.io/s/long-dust-t2lth9?file=/src/main.vue:919-925
Ticket ID: 1631556

**Comment by filipKovachev** [comment]
Reported again in Ticket ID: 1633294

**Comment by kdikov82** [comment]
The issue is observed with a single drop down as well if it is placed within initially hidden container:

- https://stackblitz.com/edit/9gsasg-hrurqg?file=src%2Fmain.vue

As a workaround the width of the popup can be set explicitly:
```
      <dropdownlist
        ...
        :popup-settings="{ width: '240px' }"
      >
```

**Comment by vveesseelliinnaa** [comment]
Reported again in Ticket ID: 1636842

**Comment by jakebarshickf1nx** [comment]
Looking forward to this being resolved!

**Comment by m-gug** [comment]
Reported again in Ticket ID: 1639917
This is a blocking bug, any ETA on this?
The workaround is no option!

**Comment by PekoPPT** [comment]
Hi, everyone.

The issue is already in our queue for fixing. The fix should be ready by the end of the current month(February 2024) and publicly available in March.

We will do our best to provide the fix even earlier but first, we need to investigate the case in detail. 

Thank you for your patience and understanding.

**Comment by zdravkov** [comment]
The issue is fixed in version 4.3.1 of Kendo UI for Vue:  https://stackblitz.com/edit/9gsasg-shmezc?file=src%2Fmain.vue
For this specific case with form in the Tab we also need to change the additional CSS used because an internal rendering change was added too. Here is the updated CSS:

`<style>
.k-tabstrip > .k-content > .k-animation-container,
.k-tabstrip > .k-tabstrip-content > .k-animation-container {
  width: 100%;
}
</style>`
